trieste/utils/objectives.py

```diff
@@ -126,6 +128,143 @@ def logarithmic_goldstein_price(x: TensorType) -> TensorType:
 """
 
 
+class MultiObjectiveTestProblem(ABC):
+    r"""Base class for test multi-objective test functions.
+    between a provided point and the closest point on the true pareto front.
+    """
+
+    @abstractmethod
+    def prepare_benchmark(self):
+        """
+        Evaluate Original problem
+        """
+
+    @abstractmethod
+    def gen_pareto_optimal_points(self, n: int):
+        """
+        Generate `n` pareto optimal points
+        """
+
+
+class VLMOP2(MultiObjectiveTestProblem):
```

Comment:
are there more descriptive names for these? Or are these well-known names?

Reply:
I think *VLMOP2* is known in multi-objective optimization community, 'V', 'L' just stands for the author's name of [1], which is a common function naming strategy in this community. 

Suggestions of a more clear naming approach is appreciated.

Reply:
As the format for these synthetic functions is different to the non-MO ones, it might be cleaner to have it as a separate file?

Reply:
It has now been moved into `mo_objectives.py`, while the test are kept in the same file.